classes/system/elasticsearch/server/cluster.yml

```diff
@@ -21,19 +21,29 @@ parameters:
   elasticsearch:
     server:
       enabled: true
-      publish_host: ${_param:single_address}
       master: true
       data: false
-      index:
-        shards: 1
-        replicas: 1
+      mlockall: true
       bind:
         address: ${_param:single_address}
         port: 9200
+      index:
+        shards: 1
+        replicas: 1
+      threadpool:
+        bulk:
+         queue_size: 1000
+      gateway:
+        expected_nodes: 3
+        recover_after_nodes: 2
+        recover_after_time: 5m
+      cors:
+        enabled: true
+        allow_origin: '/.*/'
       cluster:
         multicast: false
         minimum_master_nodes: 2
         members:
         - host: ${_param:monitor_node01_address}
         - host: ${_param:monitor_node01_address}
-        - host: ${_param:monitor_node01_address}
\ No newline at end of file
+        - host: ${_param:monitor_node01_address}
```

Comment:
I didn't start to work on cluster. So it is mostly copy paste from single for the moment. Need to be done for sure.